package.json

```diff
@@ -12,14 +12,15 @@
         "test": "nyc -r lcov perjury test/*-test.js",
         "posttest": "rm -f lib/*.js",
         "build": "coffee -b -c -o lib/ src/*.coffee",
-        "prepublish": "mkdir -p lib/ && coffee -b -c -o lib/ src/*.coffee",
+        "prepublish": "mkdir -p lib/ && npm run build",
         "postpublish": "rm lib/*.js",
         "clean": "rm -f lib/*.js *~ lib/*~ src/*~ test/*~"
     },
     "devDependencies": {
         "coffee-script": "1.6.x",
         "coveralls": "^3.0.0",
         "nyc": "^11.3.0",
-        "perjury": "^1.1.0"
+        "perjury": "^1.1.0",
+        "watch": "^1.0.2"
```

Comment:
I don't see this being used?

Reply:
It removed in rebase :/

Reply:
Done: https://github.com/e14n/ofirehose/pull/14/commits/d02f0cd98c6dfab0f07c926c1ce935b2da674445